namespaces/live-1.cloud-platform.service.justice.gov.uk/cccd-staging/resources/ecr.tf

```diff
@@ -0,0 +1,19 @@
+module "cbo_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.1"
```

Comment:
Can you please use the latest version of the module ? 

https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/tree/master/examples 

Let me know if you need help.

Reply:
Thank you, will do. When I enter github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=3.2, it is failing in circle ci with the error Cannot pass non-existent provider "eu-west-2" ( I have also tried aws.london). Can you advise?

Reply:
Hi,  thank you for using the latest version of the module. 

The issue here is that you are trying to use the`aws.london` provider to create your resources, but this provider does not exist. 

You can directly use our example/main.tf linked below, which defines the providers.

[https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/blob/master/examples/main.tf ](https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/blob/master/examples/main.tf )